### DISCUSSION TITLE
When filtering by filename, are the leading directories part of the "filename" or does filtering by filename work on only the actual filename component?

### DISCUSSION THREAD

**Discussion by tomwitt2** [Komet/MediaElch#1389] [Q&A] [answered]
While this seems a silly question, I do see the entire path listed in the "Files" entry of the Information tab.  

It would be extremely useful to me to be able to filter on the parent directory names in this full path listed in "Files".

**Comment by bugwelle** [comment, marked as answer]
Hi,

yes, as far as I can see, the full file path is used (case insensitive). Just tested it. I have a folder `elch_media` inside `Downloads`. Searching for "Downloads" yields _all_ of my movies. :-)

<img width="1039" alt="Screenshot 2022-01-21 at 13 18 32" src="https://user-images.githubusercontent.com/1667306/150526035-6fc5d581-9b87-4234-b759-17238a300132.png">


Regards,
Andre
